app/locale/en.coffee

```diff
@@ -678,6 +678,30 @@
     support_part2: "support@codecombat.com"
     support_part3: "if you have any questions."
 
+  premium_features:
+    get: "Get"
+    codecombat: "CodeCombat"
+    premium: "Premium"
```

Comment:
I think this might do better as one translation with line breaks included and setting the tag to `[html]` like with [this template](https://github.com/codecombat/codecombat/blob/master/app/templates/about.jade#L47-47). That way translators see it's one piece. Include in a comment telling translators they should make sure this text fits in the banner.